tests/test_doctestplus.py

```diff
@@ -651,3 +652,37 @@ def test_doctest_float_replacement(tmpdir):
 
     doctest.testfile(str(test2_rst), module_relative=False,
                      raise_on_error=True, verbose=False, encoding='utf-8')
+
+
+def test_doctest_subpackage_requires(testdir, caplog):
+    testdir.makeini(
+        """
+        [pytest]
+        doctest_subpackage_requires =
+            test/a/*=pytest>1
```

Comment:
Test the case of spaces around `=`?
```suggestion
            test/a/* = pytest>1
```